velox/core/Config.cpp

```diff
@@ -46,4 +48,11 @@ bool MemConfigMutable::isValueExists(const std::string& key) const {
   return lockedValues->find(key) != lockedValues->end();
 }
 
+void MemConfig::validateConfig() {
+  // Validate if timezone name can be recognized.
+  if (isValueExists(QueryConfig::kSessionTimezone)) {
```

Comment:
nit: might be better to avoid looking up the key twice

```
auto it = values_.find(key);
if (it != values_.end()) {
   ... validate it->second
}
```

Reply:
Updated to use `get` function which looks up the key once.